useful_extras/create_comparison.py

```diff
@@ -143,6 +151,10 @@ def recreate_single_figure(
                         alpha=0.3,
                         linewidth=0.0,
                     )
+ 
+                    if plot_additional:
```

Comment:
Why is this only enabled for the `shaded` version? Do we not want this in all cases?

Reply:
Because I (possibly wrongly) assumed that all median and mean lines have type `shaded`; and these are the only cases where we have additional data points to plot. For instance, mass function plots (with the type `errorbar`) will not have additional data points (for mass function lines, we currently do not save them in .yml files)

Reply:
So there are three `line_type`s that you [can choose from](https://velociraptor-python.readthedocs.io/en/latest/modules/velociraptor.autoplotter.html):

1. `shaded`: this plots the median line and shows the percentiles you choose using a shaded region
2. `errorbar`: this plots the median line and shows the percentiles using errorbars from that median line (or, if you configure the default styling for the call to `errorbar` differently, it may plot as points).
3. `none`: this plots only the line.

I think in all cases we want the ability to plot additional points.